cram-tests/file-conversion-tests.t/run.t

```diff
@@ -0,0 +1,83 @@
+Testing pdf to pdfa conversion:
+  $ ../../conversion-scripts/soffice-wrapper.sh -i pdf -o pdf < test-pdf.pdf > converted-test-pdf-to-pdfa.pdf
+  $ if [ $? -eq 0 ]; then echo "pdf to pdfa conversion was successfull"; else "pdf to pdfa conversion failed"; fi;
+  pdf to pdfa conversion was successfull
+  $ pdftotext expected-converted-test-pdf-to-pdfa.pdf
+  $ pdftotext converted-test-pdf-to-pdfa.pdf
+  $ diff expected-converted-test-pdf-to-pdfa.txt converted-test-pdf-to-pdfa.txt
```

Comment:
On Linux, I get diffs for all the plaintext conversions due to differences in where `pdftotext` chose to introduce line breaks.  I would recommend adjusting every invocation of diff so that it ignores whitespace.  I think the `-b` options and `-w` options to `diff` do different versions of whitespace ignoring; maybe try playing around with that a bit and see if you can get the diff to work on Linux as well.

(If you need a Linux system, you can try running the cram tests on the Linux cluster.  I believe they have `opam` set up there?)

Reply:
I changed the tests so that each conversion has it's own test now. 
For output pdf files generated after the conversion, when `cat`ing to terminal, cram would complain about there being a difference in output after every run. So I used pdftotext on the pdfs and used `cat` on the text files. Also, pdftotext has a 'end of line convention' option and I've set that, so the output should be same on mac and linux.